nx-dev/nx-dev/app/blog/blog-seo.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import { NextSeo } from 'next-seo';
+import { usePathname } from 'next/navigation';
+
+export default function BlogSeoComponent() {
+  return (
+    <NextSeo
+      title="Nx Blog - Updates from the Nx & Nx Cloud team"
+      description="Latest news from the Nx & Nx Cloud core team"
+      useAppDir={true}
+      openGraph={{
+        url: `https://nx.dev ${usePathname()}`,
```

Comment:
There's an extra space here. Also, is there a way to access pathname without `use client`?

Reply:
I guess it's okay since this wouldn't be that different from what we do in pages router right now.

Reply:
updated